packages/@atjson/offset-annotations/src/utils/video-urls.ts

```diff
@@ -178,11 +178,31 @@ function isBrightcoveURL(url: IUrl) {
   );
 }
 
+function isTwitchURL(url: IUrl) {
+  return (
+    url.host === "www.twitch.tv" ||
+    url.host === "m.twitch.tv" ||
+    isTwitchPlayerURL(url)
+  );
+}
+
+function isTwitchPlayerURL(url: IUrl) {
+  return url.host === "player.twitch.tv";
+}
+
+function normalizeTwitchURL(url: IUrl) {
+  let videoID = isTwitchPlayerURL(url)
+    ? getSearchParam(url.searchParams, "video")
+    : without<string>(url.pathname.split("/"), "").pop();
+  return `https://player.twitch.tv/?video=${videoID}&parent=www.example.com`;
```

Comment:
Capturing Slack convo here:
We need to additionally support channel and clip embeds
Channel: 
Embed URL: `https://player.twitch.tv/?channel=fareeha&parent=www.example.com`
Web URL: `https://www.twitch.tv/fareeha`

Clip:
Embed URL: `https://clips.twitch.tv/embed?clip=CharmingPowerfulHeronMingLee-wQMqc0rzzQW8v5Ah&parent=www.example.com`
Web URL: `https://www.twitch.tv/dangers/clip/CharmingPowerfulHeronMingLee-wQMqc0rzzQW8v5Ah`